docs/PythonAPI.md

```diff
@@ -0,0 +1,867 @@
+    NONE: ClassVar[DeviceKind] = DeviceKind.NONE
+    Signals: ClassVar[DeviceKind] = DeviceKind.Signals
+    StageAxis: ClassVar[DeviceKind] = DeviceKind.StageAxis
+    Storage: ClassVar[DeviceKind] = DeviceKind.Storage
+
+    def __init__(self, *args: None, **kwargs: Any) -> None:
+        """Initializes the DeviceKind class."""
+
+    def __eq__(self, other: object) -> bool:
+        """Checks if two DeviceKind objects are equal."""
+        
+    def __ge__(self, other: object) -> bool:
+        """Checks if this DeviceKind is greater than or equal to another."""
+        
+    def __gt__(self, other: object) -> bool:
+        """Checks if this DeviceKind is greater than another."""
+        
+    def __int__(self) -> int:
+        """Converts the DeviceKind to an integer."""
+        
+    def __le__(self, other: object) -> bool:
+        """Checks if this DeviceKind is less than or equal to another."""
+        
+    def __lt__(self, other: object) -> bool:
+        """Checks if this DeviceKind is less than another."""
+        
+    def __ne__(self, other: object) -> bool:
+        """Checks if two DeviceKind objects are not equal."""
+```
+
+- `Camera`: Class variable of `DeviceKind` that defines the cameras supported by the system. 
+
+- `NONE`: Class variable of `DeviceKind` that is set to None if no device of the specified kind is available.
+
+- `Signals`: Class variable of `DeviceKind` that defines the signals supported by the system.
+
+- `StageAxis`: Class variable of `DeviceKind` that defines the stage axes supported by the system.
+
+- `Storage`: Class variable of `DeviceKind` that defines the storage supported by the system.
+
+## Class `DeviceManager` 
+
+The `DeviceManager` class manages selection of available devices in the system.
+
+```python
+class DeviceManager:
+    def devices(self) -> List[DeviceIdentifier]:
+        """Returns a list of all available device identifiers."""
+    
+    @overload
+    def select(self, kind: DeviceKind) -> Optional[DeviceIdentifier]:
+        """Selects a specified device.
+        
+        Args:
+            kind (DeviceKind): The type of device to select.
+            
+        Returns:
+            Optional[DeviceIdentifier]: The selected device identifier, or None if no device of the specified kind is available.
+        """
+    
+    @overload
+    def select(self, kind: DeviceKind, name: Optional[str]) -> Optional[DeviceIdentifier]:
+        """Selects a specified device.
+        
+        Args:
+            kind (DeviceKind): The type of device to select.
+            name (Optional[str]): The name of the device to select.
+            
+        Returns:
+            Optional[DeviceIdentifier]: The selected device identifier, or None if the specified device is not available.
+        """
+    
+    def select_one_of(self, kind: DeviceKind, names: List[str]) -> Optional[DeviceIdentifier]:
+        """Selects one of the devices of the specified kind from the given list of names.
```

Comment:
How does it make this selection? Or put another way, what should I as the user expect when I use this method?

Reply:
Looking at: https://github.com/acquire-project/acquire-python/blob/cb2dd9bdec0f483c3391b54ef8996c570614379d/src/device_manager.rs#L69

Not sure if I am interpreting this correctly, but it seems to select the first device that fits within the specifications, so whichever one happened to be listed first.